examples/profile-createOrUpdate.js

```diff
@@ -0,0 +1,35 @@
+var myConfig = require('../myConfig');
```

Comment:
I don't see this file documented in the read me. Looks like it's a file that could be used to store creds, but do you feel we should be explaining that?

Reply:
I'll update the docs to reflect that this is the creds file and link it back to exampleConfig (which is included)